packages/terra-framework-site/src/examples/navigation-layout/HeaderExample.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link, withRouter } from 'react-router-dom';
+
+const propTypes = {
+  layoutConfig: PropTypes.object,
+};
+
+const HeaderExample = ({
+  layoutConfig,
```

Comment:
Nitpick:

If your hashes are small, I find it more readable if it's just one line.

Reply:
I agree, I'll update this.